ui/imports/shared/views/PasswordView.qml

```diff
@@ -296,14 +322,9 @@ Column {
         textField.validator: d.validator
         keepHeight: true
         textField.rightPadding: showHideConfirmIcon.width + showHideConfirmIcon.anchors.rightMargin + Style.current.padding / 2
-
-        onTextChanged: { if(textField.text.length === newPswInput.text.length) root.checkPasswordMatches() }
-        textField.onFocusChanged: {
-            // When clicking into the confirmation input, validate if new password:
-            if(textField.focus) d.passwordValidation()
```

Comment:
Removing that code the only way of validation is just when the view is ready, so when the button is enabled so, for example if the first input is too short, and you focus the second one, it does not validate so, the "too short" message is never visualized. I think that validation has to be done in 2 steps, one when user clicks on the second input, and the other, when the button is ready.

Reply:
@noeliaSD according to the new workflow:
- passwordValidation() was renamed to validatePassword() and is executed by the caller (button from the parent view)
- comparison of passwords and checking if they are too short was added to validatePassword()
As the result, all checks are done when pressing button.
Let me know if it makes sense.